pkg/kubelet/secret/secret_manager.go

```diff
@@ -201,8 +201,12 @@ type cachingSecretManager struct {
 }
 
 func NewCachingSecretManager(kubeClient clientset.Interface) (Manager, error) {
+	// TODO: Currently apiserver completely cannot keep up with 1 minute ttl in
+	// 5000-node clusters. So before we optimize apiserver and/or etcd, increase
+	// the TTL to 5 minutes (which is a lot, but...)
+	ttl := 5 * time.Minute
```

Comment:
This is not entirely true. There are different secrets for pods from different namespaces. In large tests we have many namespaces, which results that within a single node, almost every pod comes from a different namespace). That means that since we have 150.000 pods, we have roughly 150.000 "Get secret" requests per minute, which is ~2500 per second. That IS a lot.

I definitely agree that "watch for only secrets from this node" is something we should do, but I don't think this will happen in 1.6 timeframe as we have only 3.5 more weeks of coding. So we need something different for 1.6.

How about increasing it to say, 2m30s now (while figuring out a better way to handle it)?